.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Make sure you can reproduce with the latest version available**
```

Comment:
I would use a level 3 header instead (`### xyz`), or at least insert a blank line so that there is a line break.

Reply:
Github handles those nicely (i.e. the bold and description would not be on the same lines for some reasons) but still used `###` as it is cleaner